data/strings/strings.txt

```diff
@@ -13610,32 +13615,36 @@
   [error_enter_correct_facebook_page]
     tags = android
     en = Enter a valid Facebook web address, account, or page name
-	fr = Saisissez une adresse web, un compte ou un nom de page Facebook valide
+	  fr = Saisissez une adresse web, un compte ou un nom de page Facebook valide
```

Comment:
Please discard these whitespace changes here and below (or replace tabs with spaces if necessary).